libpod/container_log_linux.go

```diff
@@ -0,0 +1,114 @@
+//+build linux
+
+package libpod
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	journal "github.com/coreos/go-systemd/sdjournal"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// journaldLogOut is the journald priority signifying stdout
+	journaldLogOut = "6"
+
+	// journaldLogErr is the journald priority signifying stderr
+	journaldLogErr = "3"
+
+	// bufLen is the length of the buffer to read from a k8s-file
+	// formatted log line
+	// It consists of conmon.TSBUFLEN+2+conmon.STDIOBUFSIZE+'\0'
+	bufLen = 44 + 2 + 8192 + 1
```

Comment:
I'm a little iffy about the constants from conmon here getting out of sync...

Is there any potential harm in this being larger than the Conmon buffer? I'd be fine with throwing another 1k of RAM at this if we avoid segfaults in the future.

Reply:
that's why I want to bundle the go code with conmon so we can just import config.h and never differ ;)

as far as harm: none that I could see. I trim space and cut the buffer where it needs to be, so it should be fine being larger.